pkg/qemu/qemu.go

```diff
@@ -257,33 +258,49 @@ func Cmdline(cfg Config) (string, []string, error) {
 
 	// Network
 	args = append(args, "-netdev", fmt.Sprintf("user,id=net0,net=%s,dhcpstart=%s,hostfwd=tcp:127.0.0.1:%d-:22",
-		qemuconst.SlirpNetwork, qemuconst.SlirpIPAddress, y.SSH.LocalPort))
+		qemu.SlirpNetwork, qemu.SlirpIPAddress, y.SSH.LocalPort))
 	args = append(args, "-device", "virtio-net-pci,netdev=net0,mac="+limayaml.MACAddress(cfg.InstanceDir))
-	if len(y.Network.VDE) > 0 && !strings.Contains(string(features.NetdevHelp), "vde") {
+	usingVDE := false
+	for _, nw := range y.Networks {
+		if nw.VNL != "" {
```

Comment:
Oops, that was a thinko, the managed networks also require `netdev "vde"` support.